cfgov/v1/jinja2/v1/includes/organisms/expandable.html

```diff
@@ -39,11 +39,14 @@
      {{ 'itemscope="" itemprop="mainEntity" itemtype="https://schema.org/Question"'
         if value.is_faq else '' }} >
     <button class="o-expandable_header" type="button">
-        <span class="h4 o-expandable_label"
-              {{'itemprop="name"' if value.is_faq else ''}}>
+        <span class="{{ 'h2' if value.larger_heading else 'h4 '}} o-expandable_label"
+              {{'itemprop="name"' if value.is_faq else ''}}
+              {{ 'style="padding:15px 0;"' if value.expanded_padding else ''}}>
+            {% if value.icon %} {{ svg_icon( value.icon ) }} {% endif %} 
```

Comment:
I might tag @anselmbradford in here to see if there might be another way we want to do the added padding than a style inline.

Reply:
Yeah this should be added as an example in https://cfpb.github.io/design-system/components/expandables and the padding sorted out there. What is the use case for this again? 

We have an existing `o-expandable__padded` modifier, which should probably be merged into the default expandable (see https://github.com/cfpb/design-system/blob/abd4f9c889f3b8bf2de1a957729e54d88345b8a2/packages/cfpb-expandables/src/expandable.less#L87). That modifier could then be repurposed for this use case.

Reply:
Merging and utilizing the existing modifier is OK by me but I'm not 100% sure how to do it properly.

Reply:
https://github.com/cfpb/consumerfinance.gov/pull/8191/commits/85fcc8622e83222210f131ce0adbb771c5860e2a utilizes the o-expandable__padded modifier per https://github.com/cfpb/design-system/pull/1924 (so we should wait until that ends up in the codebase proper before merging this)